File: CppPractice/CppLeetcode/Leetcode274.cpp
class Solution {
public:
    int hIndex(vector<int>& citations) 
    {
        sort(citations.begin(),citations.end());
        reverse(citations.begin(),citations.end());
        int ans = 0;
        for(int i=0;i<citations.size();i++)
        {
            if(citations[i]>=i+1)
            {
                ans=i+1;
            }
        }
        return ans;
    }
};

File: JavaScriptPractice/Leetcode274.js
var hIndex = function(citations) 
{
    let ans=0;
    citations.sort((a,b)=>b-a);
    // citations.reverse();
    for(let i=0;i<citations.length;i++)
    {
        if(citations[i]>=i+1){ans=i+1;}
        else{break;}
    }
    return ans;
};